pkg/distributor/distributor.go

```diff
@@ -1107,6 +1109,7 @@ func (d *Distributor) metricsForLabelMatchersCommon(ctx context.Context, from, t
 		})
 	}
 	mutex.Unlock()
+	reqStats.AddFetchedSeries(uint64(len(result)))
```

Comment:
any test to be updated?

Reply:
There are no tests for the stats in other APIs.
We only test this in: https://github.com/cortexproject/cortex/blob/master/pkg/frontend/transport/handler_test.go#L57

Reply:
shall we add some?